cluster/util.py

```diff
@@ -14,9 +14,12 @@
 # along with this library; if not, write to the Free Software Foundation,
 # Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 #
+# new functions HDcentroid() by:
+#    2015 Jose Javier Garcia Aranda , Juan Ramos Diaz
 
 from __future__ import print_function
 import logging
+from HDdistances import HD_profile_dimensions
```

Comment:
Depending on how the package is installed, this will not work. Python will look for `HDistances` in the system-wide available packages and it will not be found. This is something that changed in Python 3!

To fix this, either use a relative import like:

    from .HDistances import HD_profile_dimensions

or prefix it explicitly with the package name like:

    from cluser.HDistances import HD_profile_dimensions

I think I also fixed the travis config, so you should see better output in the pull-request and you should immediately see if it successfully passes all tests in both Python 2 and 3 once you push. In order to benefit from this you should pull in the changes from my `develop` branch before pushing back. Something like:

    git pull https://github.com/exhuma/python-cluster.git develop

before pushing again. I hope it will not cause any conflicts. That depends on how long ago you last pulled from my repo.